Dockerfile

```diff
@@ -86,3 +86,14 @@ ENV FIO_CHECK_CMD /usr/bin/fiocheck
 
 # Install skopeo
 COPY --from=container-tools /skopeo/bin/skopeo /usr/bin
+
+# Install docker CLI, v20.10.14, required by the oe-builtin App preload
+RUN mkdir -p /etc/apt/keyrings \
+	&& curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg \
+	&& echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null \
+	&& apt-get update && apt-get install -y docker-ce-cli=5:20.10.14~3-0~ubuntu-focal
```

Comment:
the stuff above this clears out the apt-cache after installing things to produce a smaller image. maybe we should fold this into that logic or also clear out the apt-cache here.

Reply:
added `apt-get clean` to clean the cache.